library/Fission/CLI/Config/Connected.hs

```diff
@@ -28,6 +28,8 @@ import           Fission.CLI.Environment.Types as Environment
 import qualified Fission.CLI.Environment       as Environment
 import qualified Fission.CLI.IPFS.Connect      as Connect
 
+import qualified RIO.NonEmpty as NonEmpty
```

Comment:
Nit: for consistency let’s put this at the top of the file (general -> specific)

Reply:
:+1: